src/Presentation/AmountFormatter.php

```diff
@@ -14,8 +14,8 @@ class AmountFormatter {
 	private $locale;
 
 	private $localeData = [
-		'de_DE' => [2, ',', '' ],
-		'en_US' => [2, '.', '' ]
+		'de_DE' => [2, ',', ''],
+		'en_US' => [2, '.', '']
```

Comment:
Whaaa? I always thought that was the big difference between PHP and JS coding styles (no spaces in PHP array brackets). PHPCS doesn't complain either way. But I like the style with spaces more, so I'll change that.